record.csv

```diff
@@ -7,3 +7,4 @@ Andrew Test, Andrew Test, East, 177
 Andrew Test2, Andrew Test2, East, 178
 Andrew Test, Andrew Test, East, 178
 Andrew Test2, Andrew Test2, East, 178
+Aliena Graey, Aliena Graey, Drinkward, 111
```

Comment:
Can we please add this file to the `.gitignore` and delete it from the repo?  Or at least don't commit changes to it.

Reply:
Done